src/LodeRunner.Data/README.md

```diff
@@ -11,7 +11,26 @@
 
 ## Configuing CosmosDB Secrets for Local Use
 
-### CosmosDB Secrets
+### Using Cosmos DB Emulator
+
+If running in codespaces, the emulator should be running in devcontainer on default port `9090`. To run loderunner applications against the emaulator in a self contained development environment:
+
+- Update `LodeRunner` and `LodeRunner.API` secrets to use Cosmos Emulator running in codespaces:
+
+    > Note: Upon restart of an existing codespaces, you will be required to run this script again to make sure host files and certs are up to date.
+
+    ```bash
+        cd ../..
+        ./scripts/cosmos-emulator-mode.sh
```

Comment:
I think this should run by default and then devs can later on different cosmos collection

Reply:
we can execute it during `postStart` to set emulator as default db. @jofultz any objection?

We do notice some issues with `postStart` not kicking off with codesapces being restarted(hence the note), in which case we still need to manually run this script.

Reply:
Will hold this off until other scenarios are tested.